tests/test_account.py

```diff
@@ -125,6 +125,7 @@ def test_add_transaction_invalid_header():
     account = Account(name='Test Account')
     transaction = Transaction(date=datetime(2022, 2, 1), amount=0)
     with pytest.raises(ValueError):
+        # noinspection PyTypeChecker
```

Comment:
What are these comments for?

Reply:
It shuts up type warnings in the IDE:

![image](https://user-images.githubusercontent.com/199071/222905790-71b3c117-0f53-4df3-8ca4-c82e56cae2b6.png)

![image](https://user-images.githubusercontent.com/199071/222905803-7fc1258a-7c04-4dec-bbf2-ebbcb656a5e2.png)

Do you want me to take those out?

Reply:
I took them out :)

Reply:
Thanks. I think they're perhaps specific to your editor

Reply:
Yeah, they're IntelliJ-specific